bundler/spec/bundler/friendly_errors_spec.rb

```diff
@@ -24,7 +27,11 @@
 
       bundle :install, :env => { "DEBUG" => "true" }
 
-      expect(err).to include("Failed to load #{home(".gemrc")}")
+      if Gem::VERSION >= "3.2.0.pre.1"
+        expect(err).to include("Failed to load #{File.join(config_home, "gemrc")}")
```

Comment:
Same kind of question about the default location of `gemrc`, is it changing? If it's only being picked up if the file exists, I don't think we need all this because we are no longer creating this file for every test.

Reply:
Sorry, I'm not sure what you expected. We need to this for the old versions of RubyGems because they use only the old default location.

Can I remove this example?

Reply:
My expectation is that, since we should only create the new file structure (`~/.local/share/whatever') for the tests that need it, and those files should be cleaned up after those tests are done, this test should keep working just like it worked before in a way that is independent from the rubygems version being run.

Otherwise, when I read this test, the first thing that comes to mind is that the default `gemrc` home has changed in rubuygems 3.2, and that my gemrc file will no longer be read when I upgrade.

Reply:
I see. I will split this example to the XDG_CONFIG_HOME and old configuration.

Reply:
I added test for the old gemrc at https://github.com/rubygems/rubygems/pull/3545/commits/b27eefd51f9279fae25b64c85889472098bad4d9

But I'm still not sure what your requested. Can you show the code for this? Thanks.

Reply:
I will work on it, sure.

Reply:
But so that maybe it's more clear, my expectation is that this PR shouldn't change this test file at all. This PR deals with error reporting, and the default `.gemrc` location hasn't changed, so this PR shouldn't touch this file.